### PR TITLE
Feature author report list

### PR DESCRIPTION
**Pull request by maciejSamerdak**
Introduces list view showing all reports from selected employee

Related #130 

**Do not delete this branch until #132 is merged**

Resolves https://github.com/Code-Poets/sheetstorm/issues/90

### CODE REVIEW

File: employees/templates/employees/author_report_list.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          <thead>
+            <tr>
+                <th class="author-reports-date-header">{{ UI_text.DATE_COLUMN_HEADER.value }}</th>
+                <th class="author-reports-project-header">{{ UI_text.PROJECT_COLUMN_HEADER.value }}</th>
+                <th class="author-reports-work-hours-header">{{ UI_text.WORK_HOURS_COLUMN_HEADER.value }}</th>
+                <th class="author-reports-description-header">{{ UI_text.DESCRIPTION_COLUMN_HEADER.value }}</th>
+                <th class="author-reports-creation-date-header">{{ UI_text.CREATION_DATE_COLUMN_HEADER.value }}</th>
+                <th class="author-reports-last-update-header">{{ UI_text.LAST_UPDATE_COLUMN_HEADER.value }}</th>
+                <th class="author-reports-editable-header">{{ UI_text.EDITED_COLUMN_HEADER.value }}</th>
+                <td class="author-reports-edit-button-header Invisible"></td>
```
Comment (kbeker) [inline employees/templates/employees/author_report_list.html]: Please add TaskActivity here

File: employees/templates/employees/author_report_list.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                {% else %}
+                <tr>
+                {% endif %}
+                <td class="author-reports-project-column" >{{ report.project }}</td>
+                <td class="author-reports-work-hours-column">{{ report.work_hours_str }}</td>
+                <td class="author-reports-description-column">{{ report.description }}</td>
+                <td class="author-reports-creation-date-column">{{ report.creation_date }}</td>
+                <td class="author-reports-last-update-column">{{ report.last_update }}</td>
+                <td class="author-reports-editable-column">{{ report.editable|yesno:"False,True" }}</td>
+                <td class="author-reports-edit-button-column Invisible">
```
Comment (kbeker) [inline employees/templates/employees/author_report_list.html]: And here :)

File: employees/templates/employees/author_report_list.html (hunk 1 above)
Comment (rwrzesien) [reply employees/templates/employees/author_report_list.html]: :+1: done

File: employees/templates/employees/author_report_list.html (hunk 2 above)
Comment (rwrzesien) [reply employees/templates/employees/author_report_list.html]: :+1: done, if any of these are not OK feel free to fix them if you don't want to wait for me.

Comment (kbeker) [reply employees/templates/employees/author_report_list.html]: @rwrzesien Oki, I'll remember :)

**Pull request merged**